images/kubekins-e2e/variants.yaml

```diff
@@ -30,9 +36,3 @@ variants:
     K8S_RELEASE: stable-1.16
     BAZEL_VERSION: 2.2.0
     OLD_BAZEL_VERSION: 0.23.2
-  '1.15':
```

Comment:
are we running no more 1.15 CI?

Reply:
more specifically: this should come after 1.15 CI is shut down. is it?

Reply:
Is running https://testgrid.k8s.io/sig-release-1.15-blocking

Should not be.  Got missed a few months ago now.

Reply:
@BenTheElder -- yep, it should be shut down. 1.15 jobs have already been removed: https://github.com/kubernetes/test-infra/pull/17867

Reply:
Mostly shut down.  The dashboard @tpepper linked shows the generated jobs are still running (eg: https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/generated/generated.yaml#L1860-L1887) though I'm just now hilariously noticing that many of the jobs aren't using release-branch variants of kubekins.  Whoops.

I'm fine merging this despite that.  The jobs that use kubekins 1.15 are just going to be stuck on their current image until removed (https://github.com/kubernetes/test-infra/search?q=kubekins+1.15&unscoped_q=kubekins+1.15&type=Code)

Reply:
@spiffxp -- that's the PR to follow. Once this merges and the new kubekins-e2e images get built, I'll be able to rotate the jobs.